src/test/java/org/sonatype/repository/conan/internal/proxy/ConanProxyRecipeTest.java

```diff
@@ -29,13 +30,8 @@
 import static org.hamcrest.core.IsEqual.equalTo;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.when;
-import static org.sonatype.repository.conan.internal.proxy.ConanProxyRecipe.conanFileMatcher;
-import static org.sonatype.repository.conan.internal.proxy.ConanProxyRecipe.conanInfoMatcher;
-import static org.sonatype.repository.conan.internal.proxy.ConanProxyRecipe.conanManifestMatcher;
-import static org.sonatype.repository.conan.internal.proxy.ConanProxyRecipe.conanPackageMatcher;
-import static org.sonatype.repository.conan.internal.proxy.ConanProxyRecipe.downloadUrlsMatcher;
-import static org.sonatype.repository.conan.internal.proxy.ConanProxyRecipe.downloadUrlsPackagesMatcher;
 
+@Ignore
```

Comment:
Question: Are we ignoring because it's broke?

Reply:
Good catch (removed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/15/commits/de3298be593bd3e1f8209ef2288f59006981fffe). These tests are no longer needed because they are tested against the individual Matcher classes e.g. https://github.com/sonatype-nexus-community/nexus-repository-conan/blob/gav-ordering/src/main/java/org/sonatype/repository/conan/internal/proxy/matcher/BintrayMatcher.groovy